<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipLineEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="CelsiusCapteur1Label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="CelsiusCapteur3Label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="CelsiusCapteur2Label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Température</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CelsiusCapteur1pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Celsius Capteur 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CelsiusCapteur2pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Celsius Capteur 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CelsiusCapteur3pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Celsius Capteur 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FahrenheitCapteur1pushButton">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>230</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Fahrenheit Capteur 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FahrenheitCapteur2pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Fahrenheit Capteur 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FahrenheitCapteur3pushButton">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>330</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Fahrenheit Capteur 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HygrometrieCapteur1pushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hygrométrie Capteur 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HygrometrieCapteur2pushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hygrométie Capteur 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HygrometrieCapteur3pushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hygrométrie Capteur 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="FahrenheitCapteur1label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="FahrenheitCapteur2label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>285</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="FahrenheitCapteur3label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="HygrometrieCapteur1label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="HygrometrieCapteur2label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>280</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="HygrometrieCapteur3label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>330</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CelsiusCapteur1pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onCelsiusCapteur1ButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CelsiusCapteur2pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onCelsiusCapteur2ButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CelsiusCapteur3pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onCelsiusCapteur3ButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FahrenheitCapteur1pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onFahrenheitCapteur1ButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FahrenheitCapteur2pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onFahrenheitCapteur2ButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FahrenheitCapteur3pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onFahrenheitCapteur3ButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HygrometrieCapteur1pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onHygrometrieCapteur1ButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HygrometrieCapteur2pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onHygrometrieCapteur2ButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HygrometrieCapteur3pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onHygrometrieCapteur3ButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnectButtonClicked()</slot>
  <slot>onCelsiusCapteur1ButtonClicked()</slot>
  <slot>onCelsiusCapteur2ButtonClicked()</slot>
  <slot>onCelsiusCapteur3ButtonClicked()</slot>
  <slot>onFahrenheitCapteur1ButtonClicked()</slot>
  <slot>onFahrenheitCapteur2ButtonClicked()</slot>
  <slot>onFahrenheitCapteur3ButtonClicked()</slot>
  <slot>onHygrometrieCapteur1ButtonClicked()</slot>
  <slot>onHygrometrieCapteur2ButtonClicked()</slot>
  <slot>onHygrometrieCapteur3ButtonClicked()</slot>
  <slot>onTemperatureReadyRead()</slot>
 </slots>
</ui>
